src/main/java/com/mitsuki/jmatrix/Matrix.java

```diff
@@ -2,34 +2,36 @@
 /* --   MATRIX BUILDER    -- */
 // :: ------------------- :: //
 
-// Copyright (c) 2023 Ryuu Mitsuki
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-
-// -**- Package -**- //
+/* Copyright (c) 2023 Ryuu Mitsuki
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.mitsuki.jmatrix;
 
-// -**- Local Package -**- //
-import com.mitsuki.jmatrix.core.*;
+import com.mitsuki.jmatrix.core.IllegalMatrixSizeException;
+import com.mitsuki.jmatrix.core.InvalidIndexException;
+import com.mitsuki.jmatrix.core.JMBaseException;
+import com.mitsuki.jmatrix.core.MatrixArrayFullException;
+import com.mitsuki.jmatrix.core.NullMatrixException;
```

Comment:
Importing all classes using wildcard `*` is a bad practice. So here changed that statement by importing requirements classes one by one instead. This changes would makes:
- Code more easier to read and maintain.
- Reduce the compilation time.
- Reduce potential conflicts.